src/encoded/schemas/changelogs/annotation.md

```diff
@@ -4,6 +4,8 @@
 * Added *seqFISH* to *assay_term_name* enum.
 * Added *element gene link predictions* enum to *annotation_type* property
 * Added *ChromBPnet-model* and *BPnet-model* to *annotation_type* enum.
+* Added *element gene link predictions* enum to *annotation_type* property.
```

Comment:
Looks like there might have been an issue during rebase, this line is a repeat of line 5 above.